lib/Munin/Node/Utils.pm

```diff
@@ -20,7 +20,8 @@ sub set_difference
     my %set;
     @set{@$A} = ();
     delete @set{@$B};
-    return sort keys %set;
+    my @tmp = sort keys %set;
```

Comment:
I am not too deep into perl, thus please excuse my ignorance: wouldn't `return sort keys %set` work, as well?

Reply:
Actually yes, but https://metacpan.org/pod/release/THALJEF/Perl-Critic-1.122/lib/Perl/Critic/Policy/Subroutines/ProhibitReturnSort.pm

Reply:
Great - thank you for clarifying!